deepcell/applications/application_test.py

```diff
@@ -232,3 +242,11 @@ def test_predict_segmentation(self):
         x = np.random.rand(1, 128, 128, 1)
         y = app._predict_segmentation(x)
         self.assertEqual(x.shape, y.shape)
+
+        # test with different MPP
+        model = DummyModel()
+        app = Application(model)
+
+        x = np.random.rand(1, 128, 128, 1)
+        y = app._predict_segmentation(x, image_mpp=1.3)
+        self.assertEqual(x.shape, y.shape)
```

Comment:
This is the test that addresses the original issue, right?

Reply:
Yes, exactly